anthropic/src/models/claude.rs

```diff
@@ -4,8 +4,10 @@ use std::str::FromStr;
 
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
 pub enum ClaudeModel {
-    #[serde(rename = "claude-3-5-sonnet-20240620")]
```

Comment:
No longer being able to parse the old string value (`claude-3-5-sonnet-20240620`) sounds like a problem.

Perhaps it should remain as some other variant (`Claude35Sonnet20240620`).